encoding/thrift/inbound.go

```diff
@@ -116,11 +122,8 @@ func (t thriftOnewayHandler) HandleOneway(ctx context.Context, treq *transport.R
 func decodeRequest(
 	// call is an inboundCall populated from the transport request and context.
 	call *encodingapi.InboundCall,
-	// buf is a byte buffer from the buffer pool, that will be released back to
-	// the buffer pool by the caller after it is finished with the decoded
-	// request payload. Thrift requests read sets, maps, and lists lazilly.
-	buf *bufferpool.Buffer,
 	treq *transport.Request,
+	reader io.ReaderAt,
```

Comment:
let's keep the comment

Reply:
Do you mean comment on `buf`? 
That variable isn't present anymore, probably will move to the place using buf